index/models.py

```diff
@@ -1,73 +1,49 @@
 from django.db import models
+from django.utils import timezone
 
-class Petition(models.Model):
-	title = models.CharField(max_length=200)	#title of petition
-	description = models.CharField(max_length=4000)	#the paragraph description
-	ID = models.IntegerField(999999, primary_key=True)
-	archived = models.BooleanField(default=False)
-	hidden = models.BooleanField(default=False)
-				
-	created_date = models.DateTimeField(db_index=True, default=timezone.now)	#files the date created
-	expected_sig = models.IntegerField(300)		#the expected signature to move to the next step
-
-	author = models.ForeignKey(User, related_name='petitions')	#the author of the petition
-	tags = models.ManyToManyField(Tag, related_name='petitions')	#the tags, probably a max of 3
-	signatures = models.ManyToManyField(Signature, related_name='petitions')	#the signature
-
-	senate_response = models.ForeignKey(Responses, related_name='petitions')		#If the senate has responded , their answer
-
-	
-
-	def __unicode__(self):
-        return self.title		#returns title when asked for the item
-
-    def check_enough_sigs(self):
-    	if self.signatures.count() >= self.expected_sig:
-    		return True
-    	return False
-
-    def check_tags(self):
-    	if self.tags.count() > 3 :
-    		return False
-    	return True
-
-
-
-#creates a tag, there can be many in a single petition
+# Creates a tag, there can be at most three in a single petition
 class Tag(models.Model):
 	word = models.CharField(max_length=15, primary_key=True)	
```

Comment:
Maybe we should rename this variable to `label` instead. What do you think?

Reply:
Yeah, label would probably be a better description for the variable.